data/supporters.json

```diff
@@ -339,8 +339,8 @@
       "city": "Berlin",
       "country": "Germany",
       "link": "https://groups.google.com/forum/#!forum/creativecodeberlin",
-      "twitter": "P5Jam",
-      "contacts": [
+      "twitter": "creativeCodeBLN",
+      "contacts":[ 
```

Comment:
Thanks and welcome! Could you fix back the coding style (a space before `[`) for consistency and the beauty of code? :) I’ll merge right after.